products/green_fast_track/models/product/green_fast_track_bbls.sql

```diff
@@ -28,17 +28,12 @@ flags_ranked AS (
 flags_wide AS (
     SELECT
         {% for row in variables -%}
-            MAX(
-                CASE
-                    WHEN
-                        variable_type = '{{ row["variable_type"] }}'
-                        THEN variable_id
-                END
+            ARRAY_TO_STRING(
+                ARRAY_AGG(variable_id) FILTER (WHERE variable_type = '{{ row["variable_type"] }}'), ', '
```

Comment:
Do we want any ordering here? Either by distance(/`row_number`), or alphabetical, etc

Reply:
I'll look at some values but I think they're already ordered during the CTE above because of `OVER (PARTITION BY bbl, variable_type ORDER BY distance)`

and distance seems like a good choice

Reply:
Or just by row_number - order already computed, and in that case we're just enforcing what the postgres compiler hopefully is already doing

Reply:
looks like it was doing furthest -> nearest! and even though there's lots of ties (same lot with >1 CATs Permit), it was still deterministic

I added `order by distance asc, variable_id asc`. the `variable_id` part is to resolve distance ties and flags that where distance is null